Cargo.toml

```diff
@@ -159,16 +162,24 @@ anyhow = "1.0.69"
 derivative = "2.2.0"
 enum_dispatch = "0.3.11"
 fs_at = "0.1.6"
+futures = "0.3"
+lazy_static = "1"
```

Comment:
AFAIK lazy_static has been deprecated in favor of once_cell (which we already have as a dependency).

Reply:
https://github.com/rust-lang/rustup/pull/3367#discussion_r1560042952

Reply:
lazy_static is because this branch started before 

commit 867dfc7d46e46bc718bb4b947291990bdf4c7e61
Author: Dirkjan Ochtman <dirkjan@ochtman.nl>
Date:   Mon Nov 6 13:41:22 2023 +0100

    Replace usage of lazy_static with once_cell


fixed.